ugvc/mrd/featuremap_consensus_utils.py

```diff
@@ -0,0 +1,111 @@
+from __future__ import annotations
+
+from collections import defaultdict
+from os.path import basename
+from os.path import join as pjoin
+
+import numpy as np
+import pysam
+
+
+def write_a_pileup_record(
+    record_dict: dict, rec_id: str, out_fh: pysam.VariantFile, header: pysam.VariantHeader, min_qual: int
+):
+    info_fields = [
+        "X_QUAL",
+        "X_RN",
+    ]
```

Comment:
Please add these to the FeatureMapFields enum in ugvc/mrd/featuremap_utils.py an import them, then you can do something like 
`info_fields = [
        FeatureMapFields.X_QUAL.value,
        FeatureMapFields.X_RN.value,
    ]`

Reply:
But we are missing "X_QUAL" in FeatureMapFields(Enum)